getting_started.md

```diff
@@ -0,0 +1,104 @@
+
+![Scuttlebutt Logo](assets/hermies-300x300.png)
+
+Ok, you've been hearing about [Scuttlebutt](https://scuttlebutt.nz)  and decided that "Yes, I do want to join an amazing social network with lots of good people that no company can control and also happens to also work offline. 
+
+Here's a quick overview with the basics you need to know.
+
+First off Scuttlebutt is a protocol on which many different kinds of apps can be built. 
+
+As for the social network, there are many clients, just like there are many Twitter clients. It doesn't really matter which one you use. They're all talking on the same network. 
+
+Patchwork is the most polished client and it doesn't require you to know any geeky stuff. The only caveat is that it just wants to run on one computer, and your identity is tied to the files on the computer you install it on. There are ways around this but it's beyond the scope of this.
+
+![screenshot of Patchwork](assets/patchwork.jpg)
+
+
+## Step 0
+If you haven't done so already go to [the Scuttlebutt Home Page](README.md) and watch the video. Then come back. 
+
+## Step 1
+[Download Patchwork](http://dinosaur.is/patchwork-downloader/), and install it like any other app. 
```

Comment:
"install it like any other app" might sound really demoralizing when you download a ".AppImage" file and are really confused what to do with it, or any other operating system and get confused at what to do with what you download.

Reply:
I'm not sure what it's like on windows or linux, but on the mac it's literally just double click the thing you downloaded and drag it into Applications just like the icons show you.  It's _very_ simple and is just like any other app (that isn't from the app store). 

Is it just my experience talking that makes me think that's trivial? 
I can expand on it (double-click the .dmg, etc...) but i would need someone else to provide instructions for windows and linux installs.

Reply:
i understand it's trivial, i just mean the language used might be harmful _in the rare case where someone is confused now to install the app_. i think Windows is an .exe, which is about as common as it comes, so maybe the only confusing case is the new shiny AppImages for Linux. for macOS, do you _run_ the installer? might be worth saying, run the downloaded installer, or something. or yeah, we could give a short phrase on how to install for each os.